Sources/Core/Extensions/Other/XCTestCase+PovioKit.swift

```diff
@@ -12,9 +12,38 @@ public extension XCTestCase {
   func XCTAssertEqualURLRequest(_ lhs: URLRequest?, _ rhs: URLRequest?, file: StaticString = #filePath, line: UInt = #line) {
     XCTAssertEqual(lhs, rhs, file: file, line: line)
     guard let lhs = lhs, let rhs = rhs else { return }
-    XCTAssertEqual(lhs.url, rhs.url, "url", file: file, line: line)
-    XCTAssertEqual(lhs.httpMethod, rhs.httpMethod, "httpMethod", file: file, line: line)
-    XCTAssertEqual(lhs.httpBody ?? Data(), rhs.httpBody ?? Data(), "httpBody", file: file, line: line)
-    XCTAssertEqual(lhs.allHTTPHeaderFields ?? [:], rhs.allHTTPHeaderFields ?? [:], "allHTTPHeaderFields", file: file, line: line)
+    XCTAssertEqual(lhs.url, rhs.url, "Url doesn't equal!", file: file, line: line)
+    XCTAssertEqual(lhs.httpMethod, rhs.httpMethod, "HttpMethod doesn't equal!", file: file, line: line)
+    XCTAssertEqual(lhs.httpBody ?? Data(), rhs.httpBody ?? Data(), "HttpBody doesn't equal!", file: file, line: line)
+    XCTAssertEqual(lhs.allHTTPHeaderFields ?? [:], rhs.allHTTPHeaderFields ?? [:], "AllHTTPHeaderFields doesn't equal!", file: file, line: line)
+  }
```

Comment:
Please add a comment on my PR regarding this change, since it's still open I'm more than happy to make it more descriptive.